packages/atti-components/src/components/Slider/styles/SliderStep.js

```diff
@@ -0,0 +1,26 @@
+import styled from "styled-components";
+import SmallText from "../../Typographies/SmallText";
+import PropTypes from "prop-types";
+
+const SliderStep = styled.span`
+  ${({theme: {components: {slider: {sliderStep}}}}) => `
+    background-color: ${sliderStep.colors.backgroundColor};
+    height: ${sliderStep.sizes.height};
+    width: ${sliderStep.sizes.width};
+    
+    ${SmallText} {
+      display: flex;
+      justify-content: center;
+      margin-top: ${sliderStep.title.margins.marginTop};
+    }
+  `}
+  left: ${({leftPosition}) => leftPosition + '%'};
```

Comment:
You don't need to use a new arrow function here. Just gather the rules that use JS variables in one function to increase perfs

Reply:
+ since your `leftPosition` is not a required prop, you can end up with `left: null%` in your styles

Reply:
Done